src/stores.json

```
@@ -0,0 +1,44 @@
+[
+   {
+      "name": "United States",
+      "countryCode": "us",
```

We generally refer to these as store codes. Any reason you changed it?
```suggestion
      "storeCode": "us",
```

---

Not really, I'll refactor to `storeCode`